src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -0,0 +1,192 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{{ form_start(categoryForm) }}
+<div class="card">
+  <h3 class="card-header">
+    {{ 'Category'|trans({}, 'Admin.Catalog.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      <div class="form-group row">
+        {{ ps.label_with_help(('Name'|trans({}, 'Admin.Global')), ('Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}')) }}
+        <div class="col-sm">
+          {{ form_widget(categoryForm.name) }}
+        </div>
+      </div>
+      <div class="form-group row">
+        {{ ps.label_with_help(('Displayed'|trans({}, 'Admin.Global')), ('Click on "Displayed" to index the category on your shop.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <div class="col-sm">
+          {{ form_widget(categoryForm.active) }}
+          <small class="form-text">
+            {{ 'If you want a category to appear in the menu of your shop, go to [1]Modules > Modules & Services > Installed modules.[/1] Then, configure your menu module.'|trans({
+              '[1]': '<a href="' ~ path('admin_module_manage') ~ '" class="_blank">',
+              '[/1]': '</a>'
+            }, 'Admin.Catalog.Help')|raw }}
+          </small>
+        </div>
+      </div>
+      {% if categoryForm.id_parent is defined %}
```

Comment:
I like to have blank lines between blocks to help me read them however I am aware there are people against it. Wdyt ?